### DISCUSSION TITLE
bootstrap 5 image height - make image height in left column equal to right text column

### DISCUSSION THREAD

**Discussion by freddaw** [twbs/bootstrap#37867] [Q&A]
Using Bootstrap 5.3, I'm trying to make an image in my left column the same height as the text in the right column with the image width auto. I haven't been able to get the image to resize small enough. If I remove the image, I see left card is the same size as the right text column, but when I put the image back, it is too large, stretching the left column.

I've tried using object-fit (maybe incorrectly), with and without cards, and with h-100 on the image card. Here is my code so far:
```
<div class="row">
        <div class="col-6">
            <div class="card h-100">
              <img src="{page_image}" alt="" class="img-fluid"/>
            </div>
        </div>
       <div class="col-6">
        <div class="card">
          <div>text goes here</div>
        </div>
     </div>
   </div><!--end row-->
```

**Comment by JosephShown** [comment]
Thank you for the example, i believe the best way to do this in this case is to set a max-height property on your row and the img tag to whatever value you want. Taking your code and inserting into any online editor and loading bootstrap i then changed the code as mentioned and pictured so the column with the text which was 24px in my example then matches the column with the img which is set to that number to match the right column. Simply adjust the max-height value of the image and the row to get these to line up properly.

```
<div class="row" style="max-height: 24px;" >
        <div class="col-6">
            <div class="card h-100">
              <img src="https://via.placeholder.com/350x150" alt="" class="img-fluid" style="max-height: 24px;">
            </div>
        </div>
       <div class="col-6">
        <div class="card">
          <div>text goes here</div>
        </div>
     </div>
   </div><!--end row-->
```
